content/articles/index/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+subTitle: "# SSW Articles\n\nSSW has been developing software since 1990. Below are some of our great articles over the years. You can see much more specialized content in every area of software development on\_SSW Rules.\n"
+sidebar: |
+  <MicrosoftPanel />
+seo:
+  title: 'SSW Articles - Sydney, Melbourne, Brisbane''s Premier Software Consultants'
+  description: "SSW has been developing software since 1990. Below are some of our great articles over the years. You can see much more specialized content in every area of software development on\_SSW Rules."
+title: SSW Articles
+showMicroservices: true
+microservicesTitle: 2-Day Pre-Migration Assessment Engagement
+microservicesDescription: >-
+  Get a solid foundation for your .NET 8 migration project, ensuring you are
+  well-prepared to tackle the migration with confidence.
+microservicesUrl: /
+articles:
+  - title: Example Article
+    body: >
+      In today’s rapidly evolving tech landscape, CTOs are constantly seeking
+      ways to modernize and future-proof their applications. .NET 8 is emerging
+      as a top choice for modern app development, and here’s why.
+    pageURL: /articles/test
+    userName: Test User
+    userPosition: Software Developer
+    userImage: >-
+      https://www.ssw.com.au/people/static/Tiago-Araujo-Profile-d0d1c45c36b6f24eac389e3acc6a70ed.jpg
+  - title: Another Article
+    body: >
+      In today’s rapidly evolving tech landscape, CTOs are constantly seeking
+      ways to modernize and future-proof their applications. .NET 8 is emerging
+      as a top choice for modern app development, and here’s why.
+    pageURL: /articles/test
+    userName: Test User
+    userPosition: Test
+    userImage: >-
+      https://www.ssw.com.au/people/static/Tiago-Araujo-Profile-d0d1c45c36b6f24eac389e3acc6a70ed.jpg
```

Comment:
violates DRY - this should be sourced from the specific articles

Reply:
Done, sourcing it from content pages now

Reply:
Article content still present, not sure that this was done